wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -354,24 +476,93 @@ Winners of their respective game mode will get a Profile badge.
 
 ### osu!standard
 
+#### Round of 32
+
+Detailed results for this round are found via [this link](https://docs.google.com/spreadsheets/d/e/2PACX-1vSxQBHVfiVOXY9-vZgGyK0zEvtI20r0Ur1E_rPZBFvi9kcmiWG178l0-bosMDtxAZuusMBzfNgCYDUq/pubhtml)
+
+| Seed | Player 1 | | | Player 2 | Seed | | 
+|:---:|:---|:---:|:---:|---:|:---:|---| 
```

Comment:
```suggestion
| :-: | :-- | :-: | :-: | --: | :-: | :-: | 
```
Always have 1 space in the beginning and the end, with only 3 characters.
`| :-- |` for left align
`| :-: |` for center align
`| --: |` for right align